### DISCUSSION TITLE
Keymap not respected

### DISCUSSION THREAD

**Discussion by tommyalatalo** [LazyVim/LazyVim#162] [General]
I noticed that a couple of my keymaps aren't being respected in lazyvim, I have these:

```
-- move text up and down
keymap("n", "<C-Down>", ":m .+1<CR>==", opts)
keymap("n", "<C-Up>", ":m .-2<CR>==", opts)
```
Instead what happens when visual select and press ctrl-up or ctrl-down is that my lualine moves up and down, and I have no such keymap set in my lualine spec.

How can I go about checking and fixing this?

**Comment by folke** [comment]
Those keymaps you have are for `normal` mode, so they would not do anything in `visual` mode.

Not sure what you mean with moving lualine up and down. There's no keymaps for that....

**Comment by tommyalatalo** [comment]
Sorry, I pasted the wrong block as I have the same for several modes:
```
keymap("v", "<C-Down>", ":m '>+1<CR>gv=gv", opts)
keymap("v", "<C-Up>", ":m '<-2<CR>gv=gv", opts)
```

I'm not sure either why lualine moves up and down when I press that, but when I do then lualine jumps one step up or down depending on which arrow key I'm pressing.

**Reply by folke** [reply]
No idea. Probably a Neovim issue? Have you tried with the latest nightly? And latets stable?

Those mappings are part of LazyVim at `<A-j>` and `<A-k>`, They work fine for me in visual mode

**Reply by tommyalatalo** [reply]
That's odd... now that I tried it again then it worked fine. o.O

Something has changed for sure, because now I suddenly get tabs with multiple files again when opening another file with telescope, even though I have disabled bufferline as below: 

```
	{ "akinsho/nvim-bufferline.lua", enabled = false },
````
Do you have any idea what this could be causing this behavior?

**Reply by folke** [reply]
Bufferline changed their repo to `bufferline.nvim`

**Reply by tommyalatalo** [reply]
That was it :D
Thanks!

**Reply by tommyalatalo** [reply]
I actually have to come back to this issue as I've come upon it again.
So what's happening is that these keymaps for visual mode work as expected:
```
-- move text up and down
keymap("v", "<C-Down>", ":m '>+1<CR>gv=gv", opts)
keymap("v", "<C-Up>", ":m '<-2<CR>gv=gv", opts)
```
but the ones below for normal mode end up moving the lualine up/down one step as I mentioned before:
```
-- move text up and down
keymap("n", "<C-Down>", ":m .+1<CR>==", opts)
keymap("n", "<C-Up>", ":m .-2<CR>==", opts)
``` 
This is happening on neovim 0.8.3, and only on my lazyvim config, I'm running a 0.9.0 dev version on my work computer where this isn't happening, but is it likely that this is actually a problem in neovim itself?

**Reply by folke** [reply]
Since you can't actually move the statusline, this is defnitely a Neovim issue. But as you found, it appears to have been fixed already on Nightly. Nothing I can do about this. Use nightly to rpevent the issue, or wait till it is released as stable

**Reply by tommyalatalo** [reply]
> Since you can't actually move the statusline, this is defnitely a Neovim issue. But as you found, it appears to have been fixed already on Nightly. Nothing I can do about this. Use nightly to rpevent the issue, or wait till it is released as stable

Okay, good to know, I'll set my sights on the nvim 0.9.0 stable release then :)
